tests/install_testbed/install_neutron_node.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash -x
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FIWARE project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+# Authors: Jose Ignacio Carretero Guarde and Chema
+
+. ./config_vars
+
+INSTANCE_TUNNELS_INTERFACE_IP_ADDRESS=`ip a| awk "/inet $TUNNEL_IPS/  {print gensub(/\/../,\"\",\"g\",\\$2)}"`
+
+
+### configure rp_filter and ipforwarding.
+grep ^net.ipv4.ip_forward /etc/sysctl.conf
```

Comment:
it is not the same code that you have in install_neutron_compute_node.sh? if my question is True, why do you not provide a external script to do it and call it from these files?